__tests__/utils.test.js

```diff
@@ -74,7 +74,7 @@ describe("createRef", () => {
   });
 });
 
-describe("formatComments", () => {
+describe.skip("formatComments", () => {
```

Comment:
You shouldn't be skipping tests on PRs as this will get to the main breach - if you're looking to not run these tests when you're testing - then you can use npm test *test file name* to avoid running the utils test file